lerobot/scripts/train.py

```diff
@@ -140,24 +144,6 @@ def update_policy(
     return info
 
 
-@hydra.main(version_base="1.2", config_name="default", config_path="../configs")
```

Comment:
FYI. I nitly moved these 2 functions to be closer to `if __name__ == "__main__"`. If anyone has a problem with it, I'll move them back.